src/services/ui/src/components/AddIssueForm/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import * as UI from "@enterprise-cmcs/macpro-ux-lib";
+import { ErrorMessage } from "@hookform/error-message";
+import { CreateIssueSchema, createIssueSchema } from "shared-types";
```

Comment:
Could these two have different names? It could be confusing that they're identical except for the capitalization

Reply:
good call. Let me rework these type names